libs/api-model-bear/src/aliases.ts

```diff
@@ -45,3 +45,13 @@ export type MaqlExpression = string; // Maql expression (f.e. SELECT [/gdc/md/..
  * @public
  */
 export type BooleanAsString = "1" | "0";
+
+/**
+ * @public
+ */
+export type ThemeColor = string; // CSS color in hex format (f.g. #14b2e2)
```

Comment:
Please put the `CSS color in hex format (f.g. #14b2e2)` in the JSDoc comment. That way it will be used by IDEs to display it in IntelliSense and similar.